test/testdata/helm/charts/BUILD.bazel

```diff
@@ -1,7 +1,7 @@
 filegroup(
     name = "charts",
     srcs = glob([
-        "minecraft-3.1.4/minecraft/**",
+        "minecraft/**",
```

Comment:
question (non-blocking): Is Minecraft just the example helm chart used for testing here?

Reply:
Yes, it's a common choice for examples with both container images and Helm charts, though I don't know why that's so.